src/dispatch/static/dispatch/src/auth/store.js

```diff
@@ -14,6 +17,26 @@ const actions = {
     void state
     router.push({ path: redirectUrl.pathname })
   },
+  basicLogin({ commit }) {
+    LoginApi.login(state.creds.email, state.creds.password)
+      .then(function(res) {
+        commit("SET_USER_LOGIN", res.data.token)
+        router.push("/dashboard")
+      })
+      .catch(function() {
+        commit("app/SET_SNACKBAR", { text: "Invalid credentials", color: "red" }, { root: true })
+      })
+  },
+  register({ commit }) {
+    LoginApi.register(state.creds.email, state.creds.password)
+      .then(function(res) {
+        commit("SET_USER_LOGIN", res.data.token)
+        router.push("/dashboard")
+      })
+      .catch(function() {
+        commit("app/SET_SNACKBAR", { text: "Invalid credentials", color: "red" }, { root: true })
```

Comment:
This was supposed to bring up a snack, it doesn't and. not sure why this happens.

Reply:
That's okay, we can take a look.